web_sdk/BUILD.gn

```diff
@@ -585,6 +610,7 @@ if (!is_fuchsia) {
       ":skwasm_impl_library",
       ":skwasm_stub_library",
       ":web_engine_library",
+      ":web_locale_keymap_library",
```

Comment:
If you want to wait for my [other PR](https://github.com/flutter/engine/pull/37440), you won't need to add this value to many places. There'll be one list at the top of the file. Just add your value there and done:

https://github.com/flutter/engine/blob/5478c2f1457a5235e4bbb7ca250c0f47ad92216f/web_sdk/BUILD.gn#L31-L38